js/renovation/ui/button.tsx

```diff
@@ -79,7 +79,7 @@ export const viewFunction = (viewModel: Button): JSX.Element => {
         {ButtonTemplate && (<ButtonTemplate data={{ icon, text, ...templateData }} />)}
         {!ButtonTemplate && children}
         {isIconLeft && iconComponent}
-        {renderText && (<span className="dx-button-text">{text}</span>)}
+        {typeof renderText === 'string' && renderText && (<span className="dx-button-text">{text}</span>)}
```

Comment:
Let's not use `typeof`. Why don't we can just check that `renderText !== undefined`?

Reply:
yes, we can